CI:  E2E test case is unstable

https://github.com/apache/apisix-dashboard/runs/1377396734?check_suite_focus=true#step:5:85

```
--- FAIL: TestUpstream_Create (0.05s)
Error:     printer.go:54: PUT http://127.0.0.1:8080/apisix/admin/routes/r1
Error:     printer.go:54: PUT http://127.0.0.1:8080/apisix/admin/upstreams/1
Error:     printer.go:54: PUT http://127.0.0.1:8080/apisix/admin/routes/1
    reporter.go:23: 
        	Error Trace:	reporter.go:23
        	            				chain.go:21
        	            				response.go:585
        	            				response.go:151
        	            				base.go:155
        	            				upstream_test.go:78
        	Error:      	
        	            	expected status equal to:
        	            	 "200 OK"
        	            	
        	            	but got:
        	            	 "400 Bad Request"
        	Test:       	TestUpstream_Create
Error:     printer.go:54: GET http://127.0.0.1:9080/server_port

--- a/api/test/e2e/base.go
+++ b/api/test/e2e/base.go
@@ -94,7 +94,7 @@ func APISIXHTTPSExpect(t *testing.T) *httpexpect.Expect {
 	return e
 }
 
-var sleepTime = time.Duration(50) * time.Millisecond
+var sleepTime = time.Duration(100) * time.Millisecond
 
 type HttpTestCase struct {
 	caseDesc      string

--- a/api/test/e2e/upstream_test.go
+++ b/api/test/e2e/upstream_test.go
@@ -62,6 +62,7 @@ func TestUpstream_Create(t *testing.T) {
 			}`,
 			Headers:      map[string]string{"Authorization": token},
 			ExpectStatus: http.StatusOK,
+			Sleep:        sleepTime,
 		},
 		{
 			caseDesc:     "hit the route just created",

